Dockerfile

```diff
@@ -1,27 +1,8 @@
-# oplog-dump worker
-FROM ubuntu:14.04
+FROM google/debian:wheezy
 
-RUN apt-get -y update && apt-get install -y wget build-essential
-RUN apt-get -y update && apt-get install -y -q curl
-
-# Golang
-RUN apt-get -y update && apt-get install -y git golang bzr mercurial bash
-RUN GOPATH=/etc/go go get launchpad.net/godeb
-RUN apt-get remove -y golang golang-go golang-doc golang-src
-RUN /etc/go/bin/godeb install 1.5.1
-
-# Mongo
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
-RUN echo 'deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen' | sudo tee /etc/apt/sources.list.d/mongodb.list
-RUN apt-get -y update && apt-get install -y mongodb-org
```

Comment:
AFAICT we need to retain the installation of `mongodb-org` since we shell out to `mongodump`: https://github.com/Clever/oplog-dump/blob/master/cmd/oplog-dump/oplogdump.go#L87-L92